README.md

```
@@ -23,6 +23,21 @@
 
 ---
 
+<!-- TOC -->
+
+- [⚡ EV Charger Server](#-ev-charger-server)
+    - [🚀 Features](#-features)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)